Documentation/contributing/release/rc.rst

```diff
@@ -65,21 +65,34 @@ If you intent to release a new feature release, see the
 
    Run ``./Documentation/check-crd-compat-table.sh vX.Y``
 
-   If a branch for ``vX.Y`` doesn't exist yet, then ensure the CRD schema
-   version has not been incremented in the case there were no changes. If there
-   were changes to the CRD, then ensure it is incremented at most by 1 patch
-   version.
+   If a branch for ``vX.Y`` doesn't exist yet, then manually ensure the CRD
+   schema version has not been incremented in the case there were no changes. If
+   there were changes to the CRD, then ensure it is incremented at most by 1
+   patch version.
 
-#. Add all modified files using ``git add`` and create a pull request with the
-   title ``Prepare for release vX.Y.Z-rcW+1``. Add the backport label to the PR which
-   corresponds to the branch for which the release is being performed, e.g.
-   ``backport/1.0``.
+#. If there is a branch for this RC, prepare the changelog now.
+   If this is the first rc, ``X.Y.Z`` should be the last, stable version,
+   otherwise ``X.Y.Z`` should point to the last released RC ``contrib/release/prep-changelog.sh X.Y.Z X.Y.0-rcX``.
 
-   .. note::
+#. Add all modified files using ``git add`` and create a commit with the title
+   ``Prepare for release vX.Y.Z-rcW+1``.
+
+#. If a branch for ``vX.Y`` doesn't exist yet, we need to modify the VERSION
+   file temporarily so that ``--version`` returns the right RC version.
 
-       Make sure to create the PR against the desired stable branch. In this
-       case ``v1.0``
+   Change the file ``VERSION`` with ``vX.Y.Z-rcW+1``.
 
+#. Run ``make -C install/kubernetes all USE_DIGESTS=false``
+
+#. Add all contents in the ``install/kubernetes``, this includes the ``quick-installs`` yaml files.
```

Comment:
```suggestion
#. Add all contents in the ``install/kubernetes``, this includes the ``quick-install`` yaml files.
```

Reply:
Worth clarifying that by "add" you mean `git add` and that in later steps, it'll eventually be commited